fastcrawler/test_utils.py

```diff
@@ -0,0 +1,62 @@
+from fastcrawler.engine import AioHttpEngine
+
+
+class TestServer:
+    def __init__(self):
+        self.responses = {
+            "GET": {"message": "GET request received"},
```

Comment:
if you can, please use ENUM for http methods, you may also rework engine!

Reply:
there is a same code in the [aiohttp.hdrs](https://github.com/aio-libs/aiohttp/blob/master/aiohttp/hdrs.py)
should we rewrite ourself own ENUM?

Reply:
Of course!
we don't care about the engine we're using. It's an abstraction layer, other engine shouldn't be aware that we're using aiohttp enigne